js/views/modals/purchase/Moderators.js

```diff
@@ -60,7 +60,7 @@ export default class extends baseVw {
     const op = this.options;
     const includeString = op.include ? `&include=${op.include}` : '';
     const urlString =
-      `ob/${op.apiPath}?async=${!!op.async}${includeString}&usecache=${!!op.useCache}`;
```

Comment:
In this view, you declare defaults in the constructor. For consistency, should we follow suit with useCache?

```
const opts = {
      // set defaults
      apiPath: 'fetchprofiles',
      async: true,
      ...
}
```